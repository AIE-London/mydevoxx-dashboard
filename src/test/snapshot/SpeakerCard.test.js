/**
 * Created by DLINDSAY on 05-Apr-17.
 */
import React from 'react';
import ReactDOM from 'react-dom';
import Speaker from '../../main/components/SpeakerCard';
import renderer from 'react-test-renderer';

test('app component', () => {

    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<Speaker name="Testy MacTest" company="Testy Inc." blog="forsomereasonihaveablog.com"
                             talks={["The Importance of Jazz in modern society","Rory's Roaring Apps: An M.Night Shayamalan production","That other talk"]}/>,
                        div);
    });

});

test('reportStats component with title snapshot', () => {
    const tree = renderer.create(
        <Speaker  name="Testy" company="Testy Inc." blog="blogname.com" talks={["Talk 1 is One", "Talk 2 is Two", "Talk 3 is Three"]} />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});