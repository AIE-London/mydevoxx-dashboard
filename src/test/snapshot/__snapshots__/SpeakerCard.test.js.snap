exports[`test SpeakerCard component with title snapshot 1`] = `
<div
  className="bvnlUe row">
  <div
    className="container-fluid">
    <div
      className="row">
      <div
        className="">
        <img
          className="fOjFzA"
          src="defaultIcon.png" />
      </div>
      <div
        className="col-xs">
        <div>
          <h3>
            Testy
          </h3>
          <p>
            Testy Inc.
          </p>
        </div>
      </div>
    </div>
    <div
      className="row">
      <div
        className="col-md-3">
        <div>
          Personal Blog:
        </div>
        <div
          className="gqPtzL">
          blogname.com
        </div>
      </div>
    </div>
    <div
      className="row">
      <div
        className="col-md-3">
        <div>
          Other Talks:
        </div>
        <ul>
          <li>
            Talk 1 is One
          </li>
          <li>
            Talk 2 is Two
          </li>
          <li>
            Talk 3 is Three
          </li>
        </ul>
      </div>
    </div>
  </div>
   
</div>
`;
